HealthcareSystem/Backend/Repository/DrugInRoomRepository/IDrugInRoomRepository.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Backend.Model.Manager;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Backend.Repository.DrugInRoomRepository
+{
+   public interface IDrugInRoomRepository
+    {
+        List<DrugInRoom> GetDrugsInRoomByRoomNumber(int roomNumber);
+        DrugInRoom SetDrug(DrugInRoom drugInRoom);
+        bool DeleteDrug(int id);
+        DrugInRoom AddDrug(DrugInRoom drugInRoom);
+        DrugInRoom GetDrugInRoomByDrugId(int drugId);
+    }
+}
```

Comment:
Why we use DrugInRoom as a return type instead of the void? And why is return type bool for DeleteDrug? I think we should use the void return type and use exceptions.

Reply:
There is posibility that we will change just added objects although this disrupts the cqs.

Reply:
Return type DrugInRoom is okay but if you have time you could change it to void. But try to remove return type bool in DeleteDrug and make void return type & exceptions handling.